includes/admin/class-amp-post-meta-box.php

```diff
@@ -16,17 +16,44 @@ class AMP_Post_Meta_Box {
 	/**
 	 * Assets handle.
 	 *
+	 * @since 0.6
 	 * @const string
 	 */
 	const ASSETS_HANDLE = 'amp-post-meta-box';
 
+	/**
+	 * The post meta key.
+	 *
+	 * @since 0.6
+	 * @const string
+	 */
+	const POST_META_KEY = 'amp_status';
+
+	/**
+	 * The nonce name.
+	 *
+	 * @since 0.6
+	 * @const string
+	 */
+	const NONCE_NAME = 'amp-status';
+
+	/**
+	 * The nonce action.
+	 *
+	 * @since 0.6
+	 * @const string
```

Comment:
I think `@var` is actually what should be used here, counter-intuitively. See https://stackoverflow.com/a/18446766/93579